apps/ws/src/socket/services/ws-health-indicator.service.ts

```diff
@@ -0,0 +1,25 @@
+import { HealthIndicator, HealthIndicatorResult } from '@nestjs/terminus';
+import { Injectable } from '@nestjs/common';
+
+import { IHealthIndicator } from '@novu/application-generic';
+
+import { WSGateway } from '../ws.gateway';
+
+@Injectable()
+export class WSHealthIndicator extends HealthIndicator implements IHealthIndicator {
```

Comment:
Will rename in the upcoming PR's in order to make clear what the health indicator is responsible for.

```suggestion
export class WSServerHealthIndicator extends HealthIndicator implements IHealthIndicator {
```